### ISSUE TITLE
can't connect to certain IP address 192.168.1.13:0

### ISSUE THREAD

**Issue by willrobb2** [junyanz/interactive-deep-colorization#47]
I went to this youtube link and followed the installation instructions there. It involves docker and xquartz:

https://youtu.be/IORcb4lQlxQ

So, I follow all the instructions and this is the message I get in my terminal window instead of a GUI showing up:

Setting upsampling layer kernel: pred_313_us
ideepcolor.py: cannot connect to X server 192.168.1.13:0
wills-iMac:~ homevideostation$ 

I know that it should say mortar and pestle in the line where is says it can't connect me. Help is appreciated. I didn't bother with caffe or ubuntu or anything, I don't know if that is my problem. 
All I did was to just follow what was in the video including running docker and xquartz.

**Comment by willrobb2** [comment]
Another question. When will this become a consumer product I could download say, in the apple app store?

**Comment by junyanz** [comment]
@vbisbest

**Comment by willrobb2** [comment]
I discovered my problem. The IP address that vbisbest provided was an example in both the xhost connection and in the next instruction piece of code. I didn't notice that it was the same IP address used twice (an example IP address both times). So I replaced my own IP address not just for connecting to xhost (which I already was doing), but I used my own IP address in the final line of code that opens the GUI and it worked!
